### PR TITLE
Add smooth scroll effect on icons

### PR DESCRIPTION
**Pull request by AnshuPathak-88825**
Pr for issue #2803

### CODE REVIEW

File: documentation/src/theme/Footer/Layout/index.js [hunk 1]
```diff
@@ -178,19 +178,19 @@ export default function FooterLayout({
                                     rel="noreferrer"
                                 >
                                     {socialLink.label === "github" && (
-                                        <GithubIcon className="w-[22px] h-[22px] hover:fill-[#171515]" />
+                                        <GithubIcon className="w-[22px] h-[22px] hover:fill-[#171515] transition ease-in-out duration-500" />
```
Comment (alicanerdurmaz) [inline documentation/src/theme/Footer/Layout/index.js:181]:
```suggestion
                                        <GithubIcon className="w-[22px] h-[22px] hover:fill-[#171515] transition-colors ease-in-out duration-500" />
```

**Review by alicanerdurmaz — CHANGES_REQUESTED** [review]

**Comment by alicanerdurmaz** [comment]
Hi @AnshuPathak-88825 Thanks for the PR 🙏 

Commit message does not conform to semantic commit standards.
Can you change commit message according to [Semantic Commit Messages](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716).

For example:
feat(docs): add smooth scroll effect on icons

**Comment by AnshuPathak-88825** [comment]
done require changes

File: documentation/src/theme/Footer/Layout/index.js [hunk 2]
```diff
@@ -178,19 +178,19 @@ export default function FooterLayout({
...
-                                        <GithubIcon className="w-[22px] h-[22px] hover:fill-[#171515]" />
+                                        <GithubIcon className="w-[22px] h-[22px] hover:fill-[#171515] hover:fill-[#171515] transition-colors ease-in-out duration-500" />
                                     )}
                                     {socialLink.label === "twitter" && (
-                                        <TwitterIcon className="w-[22px] h-[22px] hover:fill-[#1DA1F2]" />
+                                        <TwitterIcon className="w-[22px] h-[22px] hover:fill-[#1DA1F2] hover:fill-[#171515] transition-colors ease-in-out duration-500" />
                                     )}
                                     {socialLink.label === "linkedin" && (
-                                        <LinkedinIcon className="w-[22px] h-[22px] hover:fill-[#0077B5]" />
+                                        <LinkedinIcon className="w-[22px] h-[22px] hover:fill-[#0077B5] hover:fill-[#171515] transition-colors ease-in-out duration-500" />
```
Comment (alicanerdurmaz) [inline documentation/src/theme/Footer/Layout/index.js:187]:
```suggestion
                                        <LinkedinIcon className="w-[22px] h-[22px] hover:fill-[#0077B5]  transition-colors ease-in-out duration-500" />
```
Do we need this ?
It doesn't cause any problems on hover effect, but it is confusing

**Review by alicanerdurmaz — CHANGES_REQUESTED** [review]

File: documentation/src/theme/Footer/Layout/index.js (hunk 2 above)
Comment (AnshuPathak-88825) [reply documentation/src/theme/Footer/Layout/index.js:187]: hover effect on icon isn't smooth that's why I made it smooth

Comment (alicanerdurmaz) [reply documentation/src/theme/Footer/Layout/index.js:187]: Transitions are okay, Thank you. 🙏 

But icons have 2 hover class right now and one is ovveriding another. This is confusing.
We should use 1 hover class on html elements.

Comment (AnshuPathak-88825) [reply documentation/src/theme/Footer/Layout/index.js:187]: ok I'm doing it now

**Comment by AnshuPathak-88825** [comment]
@alicanerdurmaz pls review it again

**Review by alicanerdurmaz — APPROVED** [review]

**Pull request merged**